Docs/Conventions/Commit_Message.md

```
@@ -0,0 +1,41 @@
+# 커밋 메시지 규칙
+
+## 구성 요소
+| 항목 | 필수 | 설명 | 예시 |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)